### PR TITLE
Cohort characterization

### PR DESCRIPTION
**Pull request by gowthamrao**
Added functionality to existing functions to make it easier to create reports. 
Feature analyses details adds details about features

### CODE REVIEW

File: R/CohortDefinition.R [hunk 1]
```diff
@@ -516,19 +516,46 @@ invokeCohortSetGeneration <- function(baseUrl, sourceKeys, definitionIds) {
...
+  })
+  inclusionRuleStatsByPersonDf <- do.call(rbind.data.frame, inclusionRuleStatsByPersonDf)
+  
+  resultsByEvent <- list(summary = summaryByEventDf, 
+                         inclusionRuleStats = inclusionRuleStatsByEventDf
+  )
+  resultsByPerson <- list(summary = summaryByPersonDf, 
+                          inclusionRuleStats = inclusionRuleStatsByPersonDf
+  )
+  results <- list(resultsByEvent = resultsByEvent, resultsByPerson = resultsByPerson
```
Comment (alondhe) [inline R/CohortDefinition.R]: To follow convention, remove the "results <-" part

File: R/WebApi.R [hunk 2]
```diff
@@ -136,6 +136,8 @@ getCdmSources <- function(baseUrl) {
     }
     data.frame(sourceName = s$sourceName,
                sourceKey = s$sourceKey,
+               sourceNameShort = gsub("\\s*\\([^\\)]+\\)","",as.character(s$sourceName)),
```
Comment (alondhe) [inline R/WebApi.R]: @gowthamrao -- I don't think these fields necessarily provide value for sites outside of Janssen. For instance, only at Janssen is there a source version (load id) in the name. 

I think we should leave these out, as they are not going to be helpful for other sites, and may instead cause confusion.

File: R/CohortDefinition.R (hunk 1 above)
Comment (gowthamrao) [reply R/CohortDefinition.R]:
```suggestion
  list(resultsByEvent = resultsByEvent, resultsByPerson = resultsByPerson
```

**Comment by schuemie** [comment]
I recommend making all changes against the develop branch, not the master branch.

**Pull request closed without merging**